02-electric/electric-section-two/electric-section-two.html

```diff
@@ -124,8 +124,8 @@ <h2 class="electric-section-two-part-two-subtitle">Savings: £51 per year</h2>
           <div class="electric-section-two-part-three-image-container">
             <img
               class="electric-section-two-part-three-image"
-              src="../../00-overall/05-images/placeholder.svg"
-              alt="image section two"
+              src="../../00-overall/05-images/02-electric/electric-page-section-two-appliances.png"
+              alt="a cartoon image of 3 kitchen appliances, a yellow microwave, a blue washing machine, and a tall red fridge"
```

Comment:
this was my mistake, change "3" to "three"

Reply:
done